data processing/file_joiner.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+# Specify the directory path
+directory_path = r'C:\Users\camil\Documents\ScoutingData'
+
+# Get a list of all ".txt" files in the directory
+json_files = [file for file in os.listdir(directory_path) if file.endswith('.json')]
+
+# Initialize an empty array
+merged_content = '['
+
+# Separator starts empty for first file
+separator = ''
+
+# Read the contents of each ".txt" file and append to the merged content
+for json_file in json_files:
+    file_path = os.path.join(directory_path, json_file)
+    with open(file_path, 'r', encoding='utf-8') as in_file:
+        merged_content += separator
+        separator = ','
+        merged_content += in_file.read()
+
+# Close the array
+merged_content += ']'
```

Comment:
What does this array declaration do?

Reply:
The merged file of all of the JSON files stores them in a list of JSON's, so the array is for the files, if you want to see what it looks like once it's been run the merged files are all in the google drive in the scouting folder